test/test_jsoo_react.re

```diff
@@ -92,6 +92,32 @@ let testOptionalProps = () => {
   });
 };
 
+let testOptionalAttributes = () => {
```

Comment:
Maybe we could call this test `testOptionalPropsLowercase` and rename the other one to `testOptionalPropsUppercase`?